fortran/nf_attio.F90

```diff
@@ -220,12 +220,15 @@ Function nf_put_att_int(ncid, varid, name, xtype, nlen, ivals) &
 
  cname = addCNullChar(name, ie)
 
-#if NF_INT_IS_C_INT 
+#if NF_INT_IS_C_INT
  cstatus = nc_put_att_int(cncid, cvarid, cname(1:ie), &
-                          cxtype, cnlen, ivals) 
-#elif NF_INT_IS_C_LONG 
+                          cxtype, cnlen, ivals)
+#elif NF_INT_IS_C_LONG
  cstatus = nc_put_att_long(cncid, cvarid, cname(1:ie), &
-                          cxtype, cnlen, ivals) 
+                          cxtype, cnlen, ivals)
+#elif NF_INT_IS_C_LONG_LONG
```

Comment:
@DennisHeimbigner this is how the magic works. The code detects that this is an i8 build and uses the longlong() function.